search/tests/test_search_result_processor.py

```diff
@@ -25,8 +25,8 @@ def test_strings_in_dictionary(self):
         })
         get_strings = SearchResultProcessor.strings_in_dictionary(test_dict)
         self.assertEqual(len(get_strings), 2)
-        self.assertEqual(get_strings[0], test_dict["a"])
```

Comment:
You're asking the wrong person. This package is simply something I had to touch to deal with the Elasticsearch upgrade. I have no idea what these tests are supposed to be doing.